spec/acceptance/tests/chocolateypackage/install_and_remove_good_package_utf-8.rb

```diff
@@ -1,12 +1,11 @@
-# require 'chocolatey_helper'
 # require 'beaker-windows'
 # test_name 'MODULES-3037 - C97738 Install known good package with utf-8 via manifest and remove via manifest'
 # confine(:to, :platform => 'windows')
 #
 # # arrange
-# package_name = '竹ChocolateyGUIÖ'
+# package_name = '???ChocolateyGUI??'
```

Comment:
we're not testing UTF-8 characters anymore?

Reply:
I guess not? The whole test is commented out in the master I'm working from, but it looks like me moving it around has wiped out the UTF-8 characters.

Reply:
Ok. I got the UTF-8 characters restored into the file. I'll take a pass at getting these test to run, but if I run into any issues, since they are commented out in the module right now, I'll take the cop out and start another ticket to get these working.